pkg/admin/zkclient.go

```diff
@@ -293,6 +295,38 @@ func (c *ZKAdminClient) GetBrokerIDs(ctx context.Context) ([]int, error) {
 	return brokerIDs, nil
 }
 
+// GetControllerID gets ID of the active controller broker
+func (c *ZKAdminClient) GetControllerID(
+	ctx context.Context,
+) (int, error) {
+	zkControllerPath := c.zNode(controllerPath)
+	log.Debugf("zookeeper controller path: %s", zkControllerPath)
+
+	var dataMap map[string]interface{}
```

Comment:
What do you think about using `zkClient.GetJson` with a well defined struct instead? Then it's clear what format the data is in. We have done this elsewhere in this file